hpobench/util/data_manager.py

```diff
@@ -926,6 +926,165 @@ def _load(self) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray, np.ndar
         return X_trn, y_trn, X_val, y_val, X_tst, y_tst
 
 
+class AdultDataManager(HoldoutDataManager):
+
+    def __init__(self):
+        super(AdultDataManager, self).__init__()
+        self.logger.debug('AdultDataManager: Starting to load data')
+        self.urls = {"data": "http://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.data",
```

Comment:
Is this a different dataset than the adult dataset on OpenML? https://www.openml.org/d/1590
If not, I would argue for using the OpenML dataset for this or is there a reason not to?

Reply:
@ayushi-3536 Can you answer this on?

Reply:
Both the datasets have same sources([UCI](https://archive.ics.uci.edu/ml/datasets/Adult) - 1996), However the one used already process a few fields and split the data into test and training sets. More info here([http://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.names). Since we are using code used in the paper for preprocessing I thought it would be safer to stick with the processed data they used.
One instance of how it is related :
-- The "Unknown" values in openml dataset are replaced with '?' in the datasets used and  in data processing code '?' replaced with nan.